GVFS/GVFS.FunctionalTests/Tools/GVFSProcess.cs

```diff
@@ -108,9 +108,12 @@ public string RunServiceVerb(string argument)
             return this.CallGVFS("service " + argument, failOnError: true);
         }
 
-        public string ReadConfig(string key)
+        public string ReadConfig(string key, bool failOnError)
         {
-            return this.CallGVFS($"config {key}", failOnError: true).TrimEnd('\r', '\n');
+            // failOnError is set to false because gvfs config read can exit with
+            // GenericError when the key-value is not available in config file. That
+            // is normal.
+            return this.CallGVFS($"config {key}", failOnError: failOnError).TrimEnd('\r', '\n');

```

Comment:
Nit: There's no need for `failOnError: ` since we're passing in the named `failOnError` now.

Reply:
Also, this comment:

```
            // failOnError is set to false because gvfs config read can exit with
            // GenericError when the key-value is not available in config file. That
            // is normal.
```

Should be moved to where `ReadConfig` is getting called.